ratatui-widgets/src/block.rs

```diff
@@ -530,6 +534,14 @@ impl<'a> Block<'a> {
         self
     }
 
+    /// Sets the block's [`MergeStrategy`] for overlapping characters. Setting it to `None`
+    /// never merges characters.
+    #[must_use = "method moves the value of self and returns the modified value"]
+    pub const fn border_merge_strategy(mut self, merge_strategy: MergeStrategy) -> Self {
```

Comment:
Simplification on naming:

```suggestion
    pub const fn merge_borders(mut self, strategy: MergeStrategy) -> Self {
```

WDYT?